pages/user/[id].vue

```diff
@@ -51,26 +51,34 @@
               @click="isEditing = true"
             >
               <EditIcon />
-              Edit
+              {{ formatMessage(messages.editButton) }}
             </button>
             <button
               v-else-if="auth.user"
               class="iconified-button"
               @click="$refs.modal_report.show()"
             >
               <ReportIcon aria-hidden="true" />
-              Report
+              {{ formatMessage(messages.profileReportButton) }}
             </button>
             <nuxt-link v-else class="iconified-button" to="/auth/sign-in">
               <ReportIcon aria-hidden="true" />
-              Report
+              {{ formatMessage(messages.profileReportButton) }}
             </nuxt-link>
           </div>
           <template v-if="isEditing">
             <div class="inputs universal-labels">
-              <label for="user-username"><span class="label__title">Username</span></label>
+              <label for="user-username"
+                ><span class="label__title">{{
+                  formatMessage(messages.profileEditUsernameLabel)
+                }}</span></label
+              >
```

Comment:
```suggestion
              <label for="user-username">
                <span class="label__title">
                  {{ formatMessage(messages.profileEditUsernameLabel) }}
                </span>
              </label>
```

In Vue 3 new lines are automatically collapsed, so this Prettier fix only creates a mess with no benefit to it.